docs/ThemePaletteDocs.md

```diff
@@ -0,0 +1,68 @@
+- Deprecate the GordonColors defined in `src/theme.ts`, which will be replaced by the new theme palettes.
+
+Throughout this project, we were loosely followed the steps described in the MUI [migration guide](https://mui.com/material-ui/experimental-api/css-theme-variables/migration/).
+
+### Changes
+
+#### Theme variables
+
+theme.ts now defines two palettes (light and dark) within a theme object using the extendTheme API.  This theme is then passed to the `Experimental_CssVarsProvider` in src/components/providers, which makes the themes accessible throughout the project via React context. [CssVarsProvider Docs](https://mui.com/material-ui/experimental-api/css-theme-variables/overview/)
+
+The theme is defined in the following object structure:
+```
+colorSchemes: {
+  light: {
+    palette: {
+      ...
+    },
+  },
+  dark: {
+    palette: {
+      ...
+    },
+  },
+},
+```
+
+Which is accessible using the CSS variable `--mui-palette` which will take on the value of light or dark palette depending on which is set(see Setting the Theme below).
+
+Within each palette there are a variety of colors, the names of which come from the Palette interface defined in mui/material/styles.  By default this includes the colors `primary`, `secondary`, `error`, `warning`, `info`, and `success`, and each of these can have `main`, `dark`, and `light`, shades, as well as shade colors 50-900 and accent colors A100-A700.  We also extend the default palette interface within theme.ts to add a neutral color, which can contain the same shades as the primary color.  [Palette Docs](https://mui.com/material-ui/customization/palette/)
+
+These palettes can also be used to override the default MUI colors for certain MUI components that don't reference the colors defined above, like cards, switches, etc.  (Some MUI components reference the colors described above by default).  For example, the color `--mui-palette-background-paper` is used by MUI cards.
+
+These colors can then be referenced using CSS variables, i.e. `var(--mui-palette-primary-main)` or `var(--mui-palette-neutral-400)` from within CSS class in .scss files.
+
+#### Setting the Theme
+
+For testing we made a component that defines a button to switch pallete modes (light, dark) with the `useColorScheme()` hook.  This sets the scheme globally, changing the color scheme of `--mui` throughout the project.
+
+#### Adding new CSS Files
+
+Gordon 360 uses CSS modules, which are specific to the folder they are in.  As part of our project to remove in-line styling where possible, we created many new CSS files.
+
+To create a new CSS for a component, create a new file in the enclosing folder called `componentName.module.scss`, and create whatever CSS classes you want.  This will create a SASS file, (so you can make use of the extra functionality SASS includes), and will also generate two files for you which convert SASS to CSS.  Then, within your component's index file, add the following line:
+
```

Comment:
Even with this good explanation, it's not clear to me why we import the css instead of scss.

Reply:
Sure, I added an extra little note that hopefully makes that clearer.